package.json

```diff
@@ -59,7 +59,9 @@
     "out/replicache.dev.wasm"
   ],
   "dependencies": {
+    "@types/lodash-es": "^4.17.4",
```

Comment:
Pulling this in for `cloneDeep`. This adds about 5kb of code to the binary :-(, but writing a deep clone -- even if restricted to json types -- has to concern itself with cycle detection. I feel like that's more your wheelhouse than mine.

Reply:
That seems fine for now.